cypress/e2e/ui-tests/test-project-contributor.cy.js

```diff
@@ -0,0 +1,93 @@
+describe("Test project contributor user functionality", () => {
+
+    // TODO: share logged in session across tests
+
+    // username here must match username in db-seed-contributor.sh
+    const username = "e2e_tests_contributor_tester"
+    const pwd = "test12345"
+
+    before(() => {
+        // seed the db with: contributor user, a blank project
+        //cy.log("Seeding the db for the contributor tests. Running db-seed-contributor.sh");
+
+        cy.log("Running seed_contributor.py")
+        cy.exec("./cypress/e2e/db-seed-contributor.sh");
+    })
+
+    it("can run the test setup", () => {
+    })
+
+    it("can create a new blank project", () => {
+        // Names of objects to create
+        const project_name = "e2e-create-proj-test";
+        const volume_name = "e2e-project-vol";
+
+        // First login as the test user
+        cy.visit("accounts/login/");
+
+        cy.get('input[name=username]').type(username);
+        cy.get('input[name=password]').type(pwd);
+
+        cy.get("button").contains('Login').click();
```

Comment:
May be add this to [beforeEach](https://docs.cypress.io/guides/references/bundled-libraries#Mocha) for these test cases?

Reply:
Absolutely. I have created a reusable cypress command using the request api in the coming version.